src/spv/spv_rpc.cpp

```diff
@@ -871,7 +1090,7 @@ UniValue spv_fundaddress(const JSONRPCRequest& request)
 
     std::string strAddress = request.params[0].get_str();
 
-    return fake_spv->SendBitcoins(pwallet, strAddress, -1);
+    return fake_spv->SendBitcoins(pwallet, strAddress, -1, 1000);
```

Comment:
We don't need to add 1000, it's just wallet value?

Reply:
In this circumstance it is never even used!

Reply:
I remember, feerate was added to SendBitcoins so spv_sendtoaddress can set feerate.